code/modules/projectiles/guns/energy/cog.dm

```diff
@@ -25,6 +25,7 @@
 	name = "\"Gear\" lasgun"
 	desc = "A Greyson Positronic design, cheap and widely produced. In the distant past - this was the main weapon of low-rank police forces, billions of copies of this gun were made. They are ubiquitous. \
 	This model has been modified by the Marshals to allow for non-lethal electrodes to be discharged as well as lasers, but at the cost of its cell-usage effifency."
+	w_class = ITEM_SIZE_NORMAL
```

Comment:
Form bulky to normal?

Reply:
This is to make it on-par with the Mamba. I could add a folding stock to it instead if preferred.

Reply:
Actually error - didn't realize I set it to normal. Amended.